docs/guides/notebooks/jupyter-notebooks/tutorial-notebooks.rst

```diff
@@ -0,0 +1,43 @@
+##################
+Tutorial Notebooks
+##################
+
+.. _NB-Intro-Use-Tutorial-NBs:
+
+How to use the Tutorial Notebooks
+=================================
+
+The best way to learn how to use a Jupyter Notebook is to open the first of the Rubin/LSST tutorial notebooks which are provided in each user's home directory.
+
+**The "notebooks/tutorial-notebooks" directory is read-only:**
+The read-only "notebooks/tutorial-notebooks" directory will *always* contain the most up-to-date versions of the tutorials.
+Notebooks can be edited and executed in this directory, but **changes cannot be saved to this directory**.
+Users wishing to edit, execute, *and save* versions of these notebooks should copy them to a different path in their home directory.
+
+**How to obtain an editable version of a tutorial notebook:**
+The commands below demonstrate how to create a copy of a tutorial notebook in the home directory which can be opened, edited, and saved.
+Step-by-step, the commands below change directory (``cd``) into the home directory (``~``),
+copy the desired tutorial to the (``cp``) into the current directory (``.``),
```

Comment:
Maybe change to:

copy (``cp``) the desired tutorial (``notebooks/tutorial-notebooks/<filename>.ipynb``) into the current directory (``.``),

?

Reply:
Yes, good call, I've implemented this in [0023f47](https://github.com/lsst/rsp_lsst_io/commit/0023f47b634c96ef2f0b9c37011ecd66a5e21d55).